src/components/Details/Content.js

```diff
@@ -0,0 +1,106 @@
+import i18n from '@dhis2/d2-i18n'
+import PropTypes from 'prop-types'
+import React from 'react'
+import css from './Details.module.css'
+import { Notice } from './Notice'
+
+export const contentItems = [
+    {
+        value: 'id',
+        label: i18n.t('Error ID:'),
+    },
+    {
+        value: 'jobId',
+        label: i18n.t('Import Job ID:'),
+    },
+    {
+        value: 'orgUnit',
+        label: i18n.t('Org Unit:'),
+    },
+    {
+        value: 'program',
+        label: i18n.t('Program:'),
+    },
+    {
+        value: 'enrollment',
+        label: i18n.t('Enrollment:'),
+    },
+    {
+        value: 'programStage',
+        label: i18n.t('Program Stage:'),
+    },
+    {
+        value: 'event',
+        label: i18n.t('Event:'),
+    },
+    {
+        value: 'tei',
+        label: i18n.t('TEI:'),
+    },
+    {
+        value: 'dataElement',
+        label: i18n.t('Data Element:'),
+    },
+    {
+        value: 'tea',
+        label: i18n.t('TEA:'),
+    },
+]
+
+const DetailsItem = ({ label, value }) => (
+    <div>
+        <span className={css.detailsItemLabel}>{label}</span>
```

Comment:
I think the most correct way to do this would be to use i18next interpolation?, e.g. `<span>{i18n.t('{{label}}: {{value}}',{label, value, nsSeparator: '-:-'})</span>` (and edit the label translations to not include the colon) (in case someone want to use something other than a colon in the target language, or the order of the label/value is reversed in the language)

That said, I don't think there's realistically any value in updating in this case. So, I'm just adding the comment as it occurred to me while looking at the PR and I'm being pedantic 🤪😔